### PR TITLE
Feature#52 deploy

### PR DESCRIPTION
**Pull request by M-Cordova-M**
# Description

We did setup and configuration to do a deploy on render.com
-modified enviroment variables
-created a script for deploy

## Screenshots

image

### CODE REVIEW

File: app/controllers/seller_controller.rb [hunk 1]
```diff
@@ -1,2 +1,30 @@
 class SellerController < OnlyphonesController
-end
+  def index
+    logger.debug "Index method called"
+    puts "Index method called"
```
Comment (JuanequeX) [inline app/controllers/seller_controller.rb:4]: remove this puts and debug

File: app/controllers/seller_controller.rb [hunk 2]
```diff
@@ -1,2 +1,30 @@
...
+    end
+  end
+  
+  def list_to_search(query="")
+    filter = query
+    request_list = request_api_filters(filter)
+    list = []
+    data_list = request_list['data']
+    data_list.each do |element|
+      puts "element: #{element}"
```
Comment (JuanequeX) [inline app/controllers/seller_controller.rb:22]: Remove this puts

File: app/controllers/seller_controller.rb [hunk 3]
```diff
@@ -1,2 +1,30 @@
...
+      id = element['id']
+      name = element['name']
+     
+      list << [id, name]
+    end
+    list
+  end
+end
```
Comment (JuanequeX) [inline app/controllers/seller_controller.rb:30]: Add a blank line at the end of this file

File: app/views/partials/_search_bar_sell.html.erb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  
+  <% puts "Entering the if block" %>
+  <% if @sell.present? %>
+  <% puts "Entering the if block" %>
+    <ul class="m-1 list-inline w-75">
+      <% @sell.each do |result| %>
+        <li class="p-2 bg-opacity-10 border rounded w-75 list-inline-item align-middle">
+        <%= puts result[0]%>
+        <a> Producto<%= result[1] %> - <%= result[0] %></a>
+        </li>
```
Comment (JuanequeX) [inline app/views/partials/_search_bar_sell.html.erb:17]: indented correctly these lines, and can you explain to me what is the reason to have some `puts` lines?

**Review by JuanequeX — CHANGES_REQUESTED** [review]

File: app/views/partials/_search_bar_sell.html.erb (hunk 4 above)
Comment (M-Cordova-M) [reply app/views/partials/_search_bar_sell.html.erb]: I wasn't getting any response from the controller and was looking for the error, but I already fixed it.

**Review by Reiyirak — APPROVED** [review]
LGTM

**Review by oscaregr — APPROVED** [review]

**Review by JuanequeX — APPROVED** [review]

**Pull request merged**